client/src/lib.rs

```diff
@@ -670,7 +625,12 @@ impl<V: vdaf::Client<16>> Client<V> {
         Error: From<<T as TryInto<Time>>::Error>,
     {
         let report = self
-            .prepare_report(measurement, &time.try_into()?)?
+            .prepare_report(
+                measurement,
+                &time.try_into()?,
+                self.leader_hpke_config.lock().await.get().await?,
+                self.helper_hpke_config.lock().await.get().await?,
```

Comment:
We could `try_join` these two for good measure.

Reply:
We can't acquire the locks within `try_join!` because the lock guard gets dropped, but we can parallelize the HTTP GET requests.